### PR TITLE
#1757 - Create common container for content in tab (Institution) - Part 2

### PR DESCRIPTION
**Pull request by sh16011993**
Used the common containers `TabContainer` and the `BodyHeaderContainer` for the institution tabs in the ministry portal. This ensures that there is no jumping effect when the ministry user clicks from one tab to the other ensuring a smooth UI transition. 

Below are the screenshots for the change:

Profile:

image

Programs:

image

Locations:

image

Users:

image

Designations:

image

Restrictions:

image

Notes:

image

I updated the styling in the `LocationSummary` component. This also affected the below view in the institution portal for the better. It basically fixed the left alignment of the Locations table with the Heading.

image

### CODE REVIEW

File: sources/packages/web/src/components/common/LocationSummary.vue [hunk 1]
```diff
@@ -2,7 +2,6 @@
   <!-- This component is shared between ministry and student users -->
   <body-header
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/components/common/LocationSummary.vue:3]: Is the `<body-header-container>` not applicable here also?

Comment (sh16011993) [reply sources/packages/web/src/components/common/LocationSummary.vue:3]: I added it in the `Locations` component, although not sure if that is the best way of doing it.

```
<template>
  <tab-container>
    <body-header-container>
      <location-summary
        :institutionId="institutionId"
        @editLocation="gotToEditLocation"
      />
    </body-header-container>
  </tab-container>
</template>
```

File: sources/packages/web/src/views/aest/institution/Designation.vue [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
-    </body-header>
-    <designation-agreement-summary
-      :designations="designations"
-      toggleMessage="No designation agreements found"
-      @viewDesignation="goToViewDesignation"
-    />
-  </full-page-container>
+  <tab-container>
+    <body-header-container>
+      <body-header
```
[the remark this answers is not in the document]
Comment (sh16011993) [reply sources/packages/web/src/views/aest/institution/Designation.vue:4]: You are correct. I will fix this.

**Review by dheepak-aot — APPROVED** [review]
LGTM. Had the same comments as @andrewsignori-aot . Please address them.

**Review by ann-aot — APPROVED** [review]
Good work @sh16011993  👍

**Review by andrepestana-aot — APPROVED** [review]
Great job!

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @sh16011993

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the changes, looks good 👍

**Pull request merged**